packages/react-components/react-infobutton/src/components/InfoButton/useInfoButtonStyles.ts

```diff
@@ -28,7 +28,7 @@ const useButtonStyles = makeStyles({
     color: tokens.colorNeutralForeground2,
 
     ...shorthands.overflow('hidden'),
-    ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStroke),
+    ...shorthands.border(tokens.strokeWidthThin, 'solid', 'transparent'),
```

Comment:
This will still have a border in Windows high contrast mode. All borders (even transparent ones) are rendered in HCM.

Reply:
Actually it looks like you're setting the border color to Canvas in forced-colors mode. It's unclear why there is a border at all?

Reply:
In WHCM outlines and box shadows are not visible, so I need to add a border in order to make the border visible. This then looks weird because if I don't add it everywhere else, the component grows in size when focused. So, I would say that I can remove it and it looks fine, but maybe make the background change color when focused in WHCM?

Reply:
Why are you using `createCustomFocusIndicatorStyle` instead of `createFocusOutlineStyle`? The latter has default styles that should work well, and renders the outline into an `::after` element to avoid affecting the layout.